test/functional/wallet_bumpfee.py

```diff
@@ -173,19 +173,50 @@ def test_feerate_args(self, rbf_node, peer_node, dest_address):
     self.sync_mempools((rbf_node, peer_node))
     assert rbfid in rbf_node.getrawmempool() and rbfid in peer_node.getrawmempool()
 
-    assert_raises_rpc_error(-8, "conf_target can't be set with fee_rate. Please provide either a confirmation target in blocks for automatic fee estimation, or an explicit fee rate.", rbf_node.bumpfee, rbfid, {"fee_rate": NORMAL, "confTarget": 1})
-
     assert_raises_rpc_error(-3, "Unexpected key totalFee", rbf_node.bumpfee, rbfid, {"totalFee": NORMAL})
-    assert_raises_rpc_error(-8, "conf_target can't be set with fee_rate. Please provide either a confirmation target in blocks for automatic fee estimation, or an explicit fee rate.", rbf_node.bumpfee, rbfid, {"fee_rate":0.00001, "confTarget": 1})
-
-    # Bumping to just above minrelay should fail to increase total fee enough, at least
-    assert_raises_rpc_error(-8, "Insufficient total fee", rbf_node.bumpfee, rbfid, {"fee_rate": INSUFFICIENT})
 
+    # For each fee mode, bumping to just above minrelay should fail to increase the total fee enough.
+    for options in [{"fee_rate": INSUFFICIENT}, {"conf_target": INSUFFICIENT, "estimate_mode": BTC_MODE}, {"conf_target": 1, "estimate_mode": SAT_MODE}]:
```

Comment:
I was originally thinking here:

> I'm confused by the ` {"conf_target": 1, "estimate_mode": SAT_MODE}`. Fitting the theme, I would expect the `conf_target` to be below `minRelayTxFeeRate`. But IIUC, 1 sat/B would be exactly the `minRelayTxFeeRate`. Is it possible that this should be … oh. … `INSUFFICIENT` also appears to be exactly the `minRelayTxFeeRate`. … Oh, this is for bumping. Duh.

Nevermind, carry on! :ok_hand: Thanks for adding this.